engine/jobmaster/dm/worker_manager.go

```diff
@@ -319,7 +328,11 @@ func (wm *WorkerManager) createWorker(
 
 func (wm *WorkerManager) stopWorker(ctx context.Context, taskID string, workerID libModel.WorkerID) error {
 	log.L().Info("start to stop worker", zap.String("task_id", taskID), zap.String("worker_id", workerID))
-	if err := wm.workerAgent.StopWorker(ctx, taskID, workerID); err != nil {
+
+	msg := &dmpkg.StopWorkerMessage{
+		Task: taskID,
+	}
+	if err := wm.messageAgent.SendMessage(ctx, taskID, dmpkg.StopWorker, msg); err != nil {
```

Comment:
We have many naming like create/delete, start/stop, resume/pause, on worker and task 😂